testing/python/issue/test_tilelang_issue_2192.py

```
@@ -0,0 +1,102 @@
+"""Regression tests for GitHub issues #2188 and #2192.
+
+#2188 – Feature: native T.named_barrier_arrive (bar.arrive) API
```

Ruff (0.15.12)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)